src/main/pg/pg_ids.h

```diff
@@ -159,7 +159,7 @@
 //#define PG_SOFTSERIAL_PIN_CONFIG    558  // removed, merged into SERIAL_PIN_CONFIG
 #define PG_GIMBAL_TRACK_CONFIG      559
 #define PG_OPTICALFLOW_CONFIG       560
-#define PG_BETAFLIGHT_END           560
+#define PG_BETAFLIGHT_END           561
```

Comment:
this value was equal to largest pgid for years (and maybe is unused). Is there any reason to change it?

Reply:
PG_BETAFLIGHT_END does not seem to be used. Should we remove it ?

Reply:
We can return it if necessary. Maybe if was used in past to avoid conflicts with cleanflight.